Update 2021-04-20-why-rust-powers-core-sdk.md (#1004)
temporalio/documentation: blog/2021-04-20-why-rust-powers-core-sdk.md

### What's complex about it?
From a 10,000 foot view, a Temporal worker follows this algorithm when running a workflow:

> 1. Long poll the server for workflow tasks (i.e., Server says "I need you to run the user's workflow code").

Reviewer: ```suggestion
1. Long poll the server for workflow tasks (for example, Server says "I need you to run the user's workflow code").
```
In this case, "e.g." would be correct, not "i.e."—but our style calls for avoiding both. See "i.e." in https://developers.google.com/style/word-list#letter-i.

Author: thanks!

We knew any design would need to meet the following requirements:

* Clean integration with other languages

> * Good ergonomics for the end user (i.e., avoid imposing new operational requirements)

Reviewer: ```suggestion
* Good ergonomics for the end user (such as avoiding new operational requirements)
```
